src/services/app-storage.ts

```diff
@@ -192,6 +192,63 @@ export class AppStorage {
 		});
 	}
 
+	/**
+	 * Delete the data for a player.
+	 *
+	 * @param {Player} player The player to delete.
+	 * @memberof AppStorage
+	 */
+	public deletePlayer(player: Player): void {
+
+		// Finds groups that deleted player is in and deletes the group
+		this.getGroups().then((groups: Group[]) => {
+
+			// Go through each group and see if the deleted player is in the group.
+			// If deleted player is in group, then delete group.
+			groups.forEach((group: Group) => {
+				group.playerIds.forEach((memberId) => {
+					if(player.storageId === memberId) {
+
+						// Delete this group because the player is being deleted
+						this.deleteGroup(group);
+					}
+				});
+			});
+
+			this.deleteObject<Player>(player, this.playersCollection);
+
+		}).catch((reason: any) => {
+			console.error(`There was an error loading the groups. Error ${reason}. Did not delete player.`);
```

Comment:
I would maybe include the player's name that was not deleted. Also this should maybe get tossed up to the next level so that it can be seen by in the UI? (maybe as a toast).